components/sidebar/navbar.tsx

```diff
@@ -50,7 +51,7 @@ export const Navbar = (props: FlexProps) => {
             <MenuList bg="white" borderColor="gray.200">
               <MenuItem>Profile</MenuItem>
               <MenuDivider />
-              <MenuItem>Sign out</MenuItem>
+              <MenuItem onClick={() => signOut()}>Sign out</MenuItem>
```

Comment:
👀

Reply:
Needed to not send the event as a payload for the logout